tests/pipeline/test_pipeline.py

```diff
@@ -2230,3 +2270,48 @@ def stateful_resource():
     assert len(fs_client.list_table_files("_dlt_loads")) == 2
     assert len(fs_client.list_table_files("_dlt_version")) == 1
     assert len(fs_client.list_table_files("_dlt_pipeline_state")) == 1
+
+
+def test_pipeline_with_frozen_schema_contract() -> None:
```

Comment:
Please reproduce a test case that user described. All starts with a database that already has a table and a pipeline in total freeze mode! then we can look for a fix in the code